### ISSUE TITLE
CSS Style broken after upgrade from 2.1.1 to 2.1.5

### ISSUE THREAD

**Issue by Zayon** [wallabag/wallabag#2694]
### Issue details

Hi,
I just did an upgrade from 2.1.1 to 2.1.5 following these instructions : https://github.com/wallabag/wallabag/issues/2639#issuecomment-262356822

It worked well but now my unread articles pages look like this : http://imgur.com/a/YtuaE

If I click on the menu (vertical 3 dots) then the display becomes correct.

### Environment

* wallabag version (or git revision) that exhibits the issue: 2.1.5
* How did you install wallabag? git clone
* Last wallabag version that did not exhibit the issue (if applicable): 2.1.1
* php version: 7.0
* OS: Ubuntu server 14.04.5 LTS
* type of hosting (shared or dedicated): dedicated
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): MySQL

### Steps to reproduce/test case

Install 2.1.1 (I migrated from 2.0.x some time ago if this matter) then follow https://github.com/wallabag/wallabag/issues/2639#issuecomment-262356822 instructions to install 2.1.5

```
rm -rf var/cache/*
git fetch origin
git fetch --tags
git checkout 2.1.5 --force
SYMFONY_ENV=prod composer install --no-dev -o --prefer-dist
php bin/console cache:clear --env=prod

**Comment by tcitworld** [comment]
Should be fixed in 2.2.

**Comment by nicosomb** [comment]
Hello @Zayon. 
I close this issue, we released 2.2 (and even 2.2.1). If this bug still happens, feel free to reopen it.
